### PR TITLE
Improvements for Docker Image docs

### PR DESCRIPTION
**Pull request by mik-laj**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: docs/docker-stack/entrypoint.rst [hunk 1]
```diff
@@ -25,68 +25,6 @@ In some cases, you can pass environment variables to the image to trigger some o
 The variables that control the "execution" behaviour start with ``_AIRFLOW`` to distinguish them
 from the variables used to build the image starting with ``AIRFLOW``.
 
-The image entrypoint works as follows:
```
Comment (mik-laj) [inline docs/docker-stack/entrypoint.rst:28]: This section contains a lot of information, which is repeated again in the sections below.

**Review by kaxil — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Build doc is failing @mik-laj

**Comment by potiuk** [comment]
Looks like some temporary problem from yesterday - rebasing and push --forceing now.

**Comment by mik-laj** [comment]
Rebased.

**Pull request merged**